files/config/default/etc/workflows/uct-publish-after-edit.xml

```diff
@@ -245,6 +245,15 @@
       </configurations>
     </operation>
 
+    <!-- Republish metadata -->
```

Comment:
Change to default "republish metadata"

Add comment that describes the problem/bug and that republishing this again fixes